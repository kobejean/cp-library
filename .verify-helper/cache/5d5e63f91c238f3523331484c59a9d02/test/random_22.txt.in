98
556345547 230293971 386296191
655719944 78497883 550831683
809639447 736894904 584148865
289498647 61848473 450413203
623422447 379686743 38436419
459485315 284096147 777961048
928286039 260655919 52791205
659271890 353662511 88353463
578792239 550802350 78686050
213878840 55346806 110966773
204733783 133551398 746832815
48888679 16512039 767615896
266115697 39031253 926422982
465270032 225762194 595900631
615845773 398366644 551314009
7464875 5899961 674323272
964549819 687340558 404300545
756471795 720642802 464388344
320911777 239735939 658004913
63106070 41442805 656636409
994890121 144358185 879542556
677545278 244124031 737832593
447599923 269286070 163343414
222230880 162741513 306960529
538478557 480389494 313175909
615277086 124455752 81072881
581286197 474115602 497575042
12401603 4839263 516401845
438416023 215944589 903644326
216836804 183488266 966574845
819132113 96162191 334561818
172711469 101243848 475614441
623803591 268710560 452296249
713699574 657695684 248807381
634141307 351424591 888506615
420859628 406916705 940087522
986374467 552785706 930251475
629049182 330938728 959987910
39417596 24729241 576700138
884874738 337818960 276190526
373697696 325093198 153452946
326110099 138915110 1377586
594648017 441147593 1827386
411991785 140024739 734353266
415842413 297800423 38460674
233478387 40414302 871203597
833008748 533496488 192686608
445219771 138333282 351522988
258197871 12227256 59113845
31625175 28626402 318296493
363230508 270902252 451285816
444995039 429494522 431140377
834488952 423304119 91738047
321848312 299064660 354767198
167721304 51024248 793233432
146819437 79962960 221062103
794868877 640593598 722057512
84816858 21275424 23535966
19018112 18831016 56860344
499090942 413793415 107491136
845707897 517650675 135199642
230947075 122202695 424150655
70555111 35922753 220013431
690276356 652214730 357499462
440599810 146712580 319953885
669263220 114746397 128429943
802771141 498540864 553500709
738908473 122378124 850294356
445698815 136937528 488469695
203265210 34896606 245357361
670051639 579368672 563806207
159235017 67892032 208135923
855752201 835782809 855752201
80244849 5571368 481469094
131630563 80202047 658152815
572788084 327728724 572788084
719951927 459218126 719951927
266081641 122188946 266081641
466340309 283784933 932680618
296287043 113034293 888861129
16029023 15366513 801451150
285887189 149072090 857661567
82660267 54159175 330641068
569384810 291249306 569384810
100817309 17218063 403269236
794225985 544166313 794225985
404998967 291202637 404998967
301365176 59708079 904095528
662388341 26100321 662388341
780485666 592582640 780485666
784950779 305961063 784950779
957803452 785454987 957803452
893264527 46047753 893264527
62828224 53781194 628282240
999999976 58823528 661764690
999999994 428571426 571428568
999999968 864406752 389830496
999999975 76923075 630769215
