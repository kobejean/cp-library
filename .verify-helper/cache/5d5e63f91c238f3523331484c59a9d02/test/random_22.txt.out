543892924
7
809639400
203572669
55
459485216
177355044
93
578792232
163089258
70
48888650
201714222
98
615845741
6323837
15
756471712
269550120
65
994890036
25798347
97
222230823
351785209
44
581286161
4155868
41
216836706
382697621
43
623803498
518117540
41
-1
45
314524590
-1
18
186848785
-1
84
137330546
-1
90
208252141
-1
26
10541711
8121985
31
278162943
149629470
27
33835
-1
100
2377165
-1
90
46189406
-1
31
88119934
19580561
2
3171260
-1
4
21614524
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
12
1
26
30
